### ISSUE TITLE
Add hint about generating hash

### ISSUE THREAD

**Issue by php-coder** [foundeo/content-security-policy.com#20]
It would be great to provide a simple instruction that people can use for generating sha256 for their code. For example:
```
$ echo -n 'box-shadow: none; border: 0px;' | openssl dgst -sha256 -binary | base64
```

**Comment by php-coder** [comment]
Or at least add this link: https://www.owasp.org/index.php/Content_Security_Policy_Cheat_Sheet#Refactoring_inline_code

**Comment by pfreitag** [comment]
Fixed in 668d9ea55ff5a623e4b0635b07a4b87c56e05ac1 see: https://content-security-policy.com/hash/
